pyms/rest_template/rest_template.py

```diff
@@ -0,0 +1,209 @@
+"""Encapsulate common rest operations between business services propagating trace headers if configured.
+"""
+import opentracing
+import requests
+from flask import current_app
+
+DATA = 'data'
```

Comment:
Consumers/Producers must to know there is a field "data" in response. It would not be better if clients does not have this mandatory. Maybe we can change signature methods "get/post/put/_for_object" for "get/post/put/_for_json" and delete the need to fetch data field.

Reply:
In Rest API design there are different opinions but some people defends using envelopes to wrap ok/error responses (for example https://medium.com/studioarmix/learn-restful-api-design-ideals-c5ec915a430f section "Use envelopes").
So both options are correct, we could choose whatever we want ;)